### ISSUE TITLE
Is there a plan to release a version corresponding to Mockito 5?

### ISSUE THREAD

**Issue by LuciferYang** [scalatest/scalatestplus-mockito#32]
It seems that Mockito 5 has become more active than Mockito 4.

**Comment by cheeseng** [comment]
Yes, the main thing with Mockito 5 is that it requires jdk11, while ScalaTest still supports jdk8.  I think we'll maintain both `mockito-4-11` and `mockito-5-6`.  I submitted the following PR: 

https://github.com/scalatest/scalatestplus-mockito/pull/33
